ocs_ci/ocs/constants.py

```diff
@@ -290,11 +286,3 @@
     'https://quay.io/api/v1/repository/rhceph-dev/ocs-registry/'
     'tag/?onlyActiveTags=true&limit=2'
 )
-
-# Platforms
```

Comment:
all these constants are in use. Any reason to delete them?

Reply:
My bad, it was old repo, corrected in pr785